docs/rules/no-restricted-browser-globals-during-ssr.md

```diff
@@ -0,0 +1,32 @@
+# Disallow access to global browser APIs during SSR (`lwc/no-restricted-browser-globals-during-ssr`)
```

Comment:
What is the value of the `lwc/no-document-during-ssr` rule if the `lwc/no-restricted-browser-globals-during-ssr` exists?

Reply:
Removed `lwc/no-document-during-ssr`.